katcp/resource_client.py

```diff
@@ -554,53 +554,53 @@ def _request_factory(self, name, description):
 
     @tornado.gen.coroutine
     def _inspecting_client_state_callback(self, state, model_changes):
-        log.debug('{}: Received {}, {}'
+        self._logger.debug('{}: Received {}, {}'
                   .format(self.address_string, state, model_changes))
         if state.connected:
             if not state.synced:
-                log.debug('{}: Setting state to "syncing"'.format(self.address_string))
+                self._logger.debug('{}: Setting state to "syncing"'.format(self.address_string))
                 self._state.set_state('syncing')
                 if model_changes:
-                    log.debug('{}: handling model updates: {}'.format(
+                    self._logger.debug('{}: handling model updates: {}'.format(
                         self.address_string,model_changes))
                     yield self._update_model(model_changes)
-                    log.debug('{}: finished handling model updates'
+                    self._logger.debug('{}: finished handling model updates'
```

Comment:
FYI: The indentation of lines 558, 574, 577 and a few more are now out. 
We should really consider, at some point, improving the debug logging in katcp by using the more clunky args style and save a few clock cycles with all these debug messages.